### ISSUE TITLE
Show exposed type value and list of values for exposed type

### ISSUE THREAD

**Issue by FlorianJacta** [Avaiga/taipy-studio-config#46]
**Description**
The goal of this issue would be to always show the exposed type used for the Data node (Tabular Data nodes). We should also be able to see the list of predefined exposed types (pandas, modin) when we edit this property in order to change it easily without going in the documentation.

**Acceptance Criteria**
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Propagate any change on the demos and run all of them to ensure there is no breaking change
- [ ] Ensure any change is well documented

**Comment by FredLL-Avaiga** [comment]
I guess we'll have to add a non-restrictive enum to the exposed_type property @jrobinAV ?
What would be the values then ?
Is there a different set of value depending on the storage_type value ?
From the description, I guess the value could also be a user defined python class ?
"enum": [
"modin",
"numpy",
"pandas" (default),
""
]

**Comment by FredLL-Avaiga** [comment]
Does "always show the exposed type" mean that you want it to be shown by default in the details view of config @FlorianJacta ?
Should the default value be explicitly "dataframe" or empty ?

**Comment by FlorianJacta** [comment]
Yes, that's it, in the details. I think it is an important property for this kind of Data nodes and it might make people understand how from a CSV we have a Pandas dataframe. It is also an essential information for the Data node. The list is 'pandas", 'modin', 'numpy'.

The default property should be 'pandas' but it also depends on the Default configuration of Data nodes of the user. So, I don't know how we can handle this. Am I right @jrobinAV ?

**Comment by jrobinAV** [comment]
Yes indeed, we should add the enum values into the schema.
